docs/working-with-data.md

```diff
@@ -0,0 +1,43 @@
+# Working with Data
+
+The BC Data Service (formerly known as Data BC) encourages and enables the strategic management and sharing of data across the government enterprise and with the public. 
+
+Access to trusted, high quality data is essential for the success of British Columbia - expressed in effective government decision making, a robust economy, and improved well-being for all of its citizens. 
+
+## API Resources
+
+### API Directory
+
+The [API Directory](https://api.gov.bc.ca/devportal/api-directory) provides a place to share your API for developers to discover.
+
+### API Management Platform
+
+The BC Government [API Management Platform](docs/default/component/aps-infra-platform-docs) is a common component that serves as a central entry point for APIs, and acts as an intermediary between applications and backend services. 
+
+### API Registry
+
+APIs within the [BC Data Catalogue](#bc-data-catalogue) can be found in the [API Registry](https://catalogue.data.gov.bc.ca/group/bc-government-api-registry)
+
+### API Services Portal
+
+If you are a developer or part of a government team, you can administer and manage your APIs using the [API Services Portal](https://api.gov.bc.ca/).
+
+## Other Resources
+
+### BC Data Catalogue
+
+[The BC Data Catalogue](https://www2.gov.bc.ca/gov/content/data/bc-data-catalogue) provides the easiest access to government's data holdings, as well as applications and web services. Thousands of the datasets discoverable in the Catalogue are available under the Open Government License - British Columbia.
```

Comment:
Perhaps add a link for Open Government Licence - British Columbia (https://www2.gov.bc.ca/gov/content/data/open-data/open-government-licence-bc).  Also note spelling of "Licence" - nonUS version.

Also, I'd change "as well as applications and web services" to "as well as their associated applications and web services". The intent is to describe data holdings as the primary entries, and apps and web services as resources attached to those entries.